sql/base/mangos.sql

```diff
@@ -1219,7 +1219,6 @@ CREATE TABLE `creature_template` (
   `SubName` char(100) DEFAULT NULL,
   `MinLevel` tinyint(3) unsigned NOT NULL DEFAULT '1',
   `MaxLevel` tinyint(3) unsigned NOT NULL DEFAULT '1',
-  `HeroicEntry` mediumint(8) unsigned NOT NULL DEFAULT '0',
```

Comment:
I kinda snuck this change in here because it currently breaks building/updating the DB.

Reply:
Is that or fixing the query in line 1325

Reply:
yes